### DISCUSSION TITLE
How can I delete a set of values ?

### DISCUSSION THREAD

**Discussion by pullmann4rent** [vitaly-t/pg-promise#900] [Announcements]
### Expected behavior

Hello,

pgp.helpers gives me insert and update but I cant see the delete function.

How can I delete an array of ids with pg-promise ?

` const query = await t.none('DELETE FROM product_media WHERE id IN ...');`

or how can I execute this query now ?

```
    const query = await t.none('DELETE FROM product_media WHERE id IN ($<values:raw>)');

        const cs = pgp.helpers.values(imgToDelete, ['id']);

        const cs_v = t.none
```

**Comment by vitaly-t** [comment]
[`:csv` filter does that](https://github.com/vitaly-t/pg-promise#csv-filter)
